From 1833fcfe449afe62ac873b7af5319ef2ddcb4726 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Tue, 8 Jan 2013 16:31:09 +0800
Subject: [PATCH 382/505] ENGR00239230 VPU: Fix a build break for Android

This patch fixes a build break for Android:
==========================================
/home/b17645/repos/AndroidJB/prebuilts/gcc/linux-x86/
arm/arm-linux-androideabi-4.6/bin/../lib/gcc/
arm-linux-androideabi/4.6.x-google/../../../../
arm-linux-androideabi/bin/ld:
out/target/product/sabresd_6dq/obj/EXECUTABLES/
mxc-vpu-test_intermediates/main.o: in function main:
external/linux-test/test/mxc_vpu_test/main.c:558:
error: undefined reference to 'transcode_test'
collect2: ld returned 1 exit status
make: *** [out/target/product/sabresd_6dq/obj/
EXECUTABLES/mxc-vpu-test_intermediates/LINKED/
mxc-vpu-test] Error 1
==========================================

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_vpu_test/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_vpu_test/Android.mk b/test/mxc_vpu_test/Android.mk
index f97e00e..dcafa7f 100644
--- a/test/mxc_vpu_test/Android.mk
+++ b/test/mxc_vpu_test/Android.mk
@@ -8,6 +8,7 @@ LOCAL_SRC_FILES := \
        enc.c \
        fb.c \
        loopback.c \
+       transcode.c \
        utils.c \
        main.c
 
-- 
1.8.0

